app.js

```diff
@@ -245,8 +245,24 @@ app.runOnStartup = function() {
 		utils.refreshExchangeRates();
 	}
 
+	if (global.totalCoinSupply == null) {
```

Comment:
Isn't safer to use `===` (strict comparison) rather than `==` ?

Reply:
Non-strict comparison covers all false-like values, e.g.:
```
null
undefined
NaN
empty string ("")
false
```
And that's what we want to achieve. I do agree however, that writing `value == null` is not too elegant, so I'll change it to `!value`